front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -352,27 +352,29 @@
               align-items: center;
               justify-content: right;
 
-              .after-midnight {
-                max-width: 1rem;
-                margin: 0 0.5rem;
-              }
-
-              .scenario-timetable-train-departure {
-                font-weight: 400;
-                font-size: 1rem;
-              }
-
-              .not-honored-or-too-fast {
+              .logo {
```

Comment:
Naming nit: maybe "indicator" would be slightly better? Or "status-icon"?

I'm completely fine with keeping the current name, feel free to ignore.

Reply:
Done